apps/web/pages/500.tsx

```diff
@@ -1,11 +1,13 @@
 import Head from "next/head";
+import { useRouter } from "next/router";
 
 import { APP_NAME, WEBSITE_URL } from "@calcom/lib/constants";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { Button } from "@calcom/ui";
 
 export default function Error500() {
   const { t } = useLocale();
+  const router = useRouter();
 
   return (
     <div className="flex h-screen">
```

Comment:
NIT: unrelated but i think this container isnt mobile responsive

Reply:
Indeed, I drafted this PR because mobile was pending testing/tweaking, it will work as it works for the booking page.